transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemAdd.java

```diff
@@ -473,6 +474,14 @@ private void performRecoveryEnvBoottime(OperationContext context, ModelNode mode
                 .addService(TxnServices.JBOSS_TXN_EXTENDED_JBOSS_XA_TERMINATOR, extendedJBossXATerminatorService)
                 .setInitialMode(Mode.ACTIVE).install();
 
+        final JBossContextXATerminatorService contextXATerminatorService = new JBossContextXATerminatorService();
+        context.getServiceTarget()
+                .addService(TxnServices.JBOSS_TXN_CONTEXT_XA_TERMINATOR, contextXATerminatorService)
```

Comment:
I hope I address this question with the one above. The point is that wfly txn client delegate call of context xa terminator to narayana xa terminator at the end. So client needs to know the instance of the terminator to delegate to.